src/components/UI/Button/button.story.tsx

```diff
@@ -18,6 +22,17 @@ export const button: Story = {
   args: {
     children: 'Click me!',
   },
+  play: async ({ canvasElement, args }) => {
+    const onClickMock = args.onClick as Mock;
```

Comment:
You are not mocking anything, just telling ts to declare the onclick method as Mock. 
Please use mocking function for this

Reply:
I see. I didn't realize the mocking function needed to be called. I tried to revise in the [commit](https://github.com/Deadlink-Hunter/Broken-Link-Website/pull/156/commits/0f1116092b0975bc6789d5a3e69fe354c349140b)